OpenRA.Mods.Common/Traits/CustomSelectionSize.cs

```diff
@@ -13,9 +13,7 @@
 
 namespace OpenRA.Mods.Common.Traits
 {
-	[Desc("Special case trait for unselectable actors that need to define targetable area bounds",
-	"for special cases like C4, engineer repair and tooltips.",
-	"Examples: bridge huts and crates.")]
+	[Desc("Special case trait for actors that need to define targetable area and rendering bounds manually.")]
```

Comment:
That is what I thought as well until I was lectured by @pchote in https://github.com/OpenRA/OpenRA/pull/9203 so I wanted to share that knowledge.